packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -443,8 +423,12 @@ export function TransferPanelSummary({
         <div className="flex flex-col space-y-2 text-sm text-gray-dark lg:text-base">
           <div className="flex flex-row justify-between">
             <div className="flex flex-row items-center space-x-2">
-              <span className="pl-4 font-light">{parentLayer} gas</span>
-              <Tooltip content={layerGasFeeTooltipContent(parentLayer)}>
+              <span className="pl-4 font-light">
```

Comment:
not sure if we can do source/destination here because gas is strictly L1/L2, so we should do parent/chain

Reply:
updated